.travis.yml

```diff
@@ -20,11 +17,14 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=4.0-DEVELOPMENT-SNAPSHOT-2017-06-11-a
+      env: SWIFT_SNAPSHOT=4.0-DEVELOPMENT-SNAPSHOT-2017-07-05-a
```

Comment:
Probably not a big deal, but just in case they introduce/revert a breaking change in between like the issue with tuples.